src/python/gudhi/datasets/remote.py

```diff
@@ -0,0 +1,199 @@
+# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+# Author(s):       Hind Montassif
+#
+# Copyright (C) 2021 Inria
+#
+# Modification(s):
+#   - YYYY/MM Author: Description of the modification
+
+from os.path import join, exists, expanduser
+from os import makedirs
+
+from urllib.request import urlretrieve
+import hashlib
+import shutil
+
+import numpy as np
+
+def get_data_home(data_home = None):
+    """
+    Return the path of the remote datasets directory.
+    This folder is used to store remotely fetched datasets.
+    By default the datasets directory is set to a folder named 'remote_datasets' in the user home folder.
+    Alternatively, it can be set by giving an explicit folder path. The '~' symbol is expanded to the user home folder.
+    If the folder does not already exist, it is automatically created.
+
+    Parameters
+    ----------
+    data_home : string
+        The path to remote datasets directory. Default is `None`, meaning that the data home directory will be set to "~/remote_datasets".
```

Comment:
If I have cloned gudhi-data locally, can I pass that directory and will it just work?

Reply:
Yes I suppose, since it works with any given directory.

Reply:
I will have to be careful to pass `/path/to/checkout/points` and not just `/path/to/checkout` otherwise the paths won't match.

Reply:
Or we could add `points/` in the paths for the cache, so that a checkout of gudhi-data as `~/gudhi_data` would work.

Reply:
Ah I see what you mean now. Well actually, giving any other different name as a directory will put everything directly there, considering that the user wouldn't want the program to add subfolders to his path because he went to the trouble of specifying something else. And since the fetch is done separately anyway using different functions for bunny, spiral_2d, or whatever, the directory, if specified, should not be altered I think, and the user should set the full path for each dataset.
To be consistent with gudhi-data, I added `points/` when fetching bunny and spiral_2d using the default directory.